### ISSUE TITLE
Associating images and notes in a relay to use nostr as diary

### ISSUE THREAD

**Issue by xeruf** [nostr-protocol/nostr#176]
TLDR: Save media files next to notes on a realy for posterity.

I started setting up an isolated nostr instance for sharing updates with friends and family including personal photos.
I do not want to upload that media to a third-party server for multiple reasons:
- privacy
- not wanting to rely on an external service for what is supposed to be a personal archive/diary of some sort

I am trying to bridge the gap between personal journaling tools like jrnl and org-journal I am using and messages - I find myself from time to time  entering the same things into my journal that I am sharing with friends, and I hope nostr could be a way to avoid duplication and associate media with the post as well.

So for maximum compatibility I was thinking of a relay implementation that would two-way-sync with an existing format like jrnl or org-journal (the latter has markup so might be preferred) and serve media linked with a relative path, so all needed information can be stored side-by-side.
The idea here is that the media is served at the path it actually resides on the server, so that even if that software goes away any old http file server could substitute, or a conversion software could export it because all needed files are there in one bundle. On the other hand you could just as easily develop a CDN that intercepts the filepaths and serves the files in some accelerated way rather than directly from the directory. But either way, you wouldn't rely on a particular software to keep your memories intact.
Thoughts?

**Comment by fiatjaf** [comment]
Not a bad idea. Let's think about this.

**Comment by everlee78** [comment]
notes.blockcore.net
